src/Components/Steps/HouseholdMembers/HHMSummaryCards.tsx

```diff
@@ -20,7 +21,7 @@ type HHMSummariesProps = {
 };
 
 const HHMSummaries = ({ activeMemberData, triggerValidation, questionName }: HHMSummariesProps) => {
-  const { formData, whiteLabel } = useContext(Context);
+  const { formData, whiteLabel } = useContext(Context) as WrapperContext;
```

Comment:
I don't think that you need this.
```suggestion
  const { formData, whiteLabel } = useContext(Context);
```